tools/mo/requirements.txt

```diff
@@ -1,6 +1,7 @@
+-c ../constraints.txt
 numpy>=1.16.6
-networkx~=2.5; python_version <= "3.6"
-networkx<=2.8.8; python_version > "3.6"
-defusedxml>=0.7.1
-importlib-metadata; python_version < "3.8" and sys_platform == 'win32'
-openvino-telemetry>=2022.1.0
\ No newline at end of file
+singledispatchmethod
```

Comment:
please remove - not needed for production

Reply:
I reverted it because it was a debug commit - however, @jiwaszki do we need `singledispatchmethod` in `tools`?

Reply:
Agreed with @jiwaszki offline to consider doing it in a separate PR. The package is currently removed, as you @rkazants requested.